### PR TITLE
feat: APPS-2799 Create Components FTVA BlockScreeningDetail & SectionScreeningDetails

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-2799](https://jira.library.ucla.edu/browse/APPS-2799) | [ISSUE-563](https://github.com/UCLALibrary/ucla-library-website-components/issues/553)

**Notes**
+ VideoEmbed to size as screen gets smaller
+ Will the DividerWay be contained on the page?

---

### View Deployments
+ **BlockScreeningDetail**: https://deploy-preview-565--ucla-library-storybook.netlify.app/?path=/story/block-screening-detail--default
+ **SectionScreeningDetails**: https://deploy-preview-565--ucla-library-storybook.netlify.app/?path=/story/section-screening-details--multiple-screenings

---

**Component Created:** 
+ `BlockScreeningDetail.vue`
+ `SectionScreeningDetails.vue`

**Stories:** 
+ `~/stories/BlockScreeningDetail.stories.js`
+ `~/stories/SectionScreeningDetails.stories.js`

**Spec:** 
+ `~/stories/BlockScreeningDetail.spec.js` 
+ `~/stories/SectionScreeningDetails.spec.js`

**Checklist:**

-   [x] I checked that it is working locally in the dev server
-   [x] I checked that it is working locally in the storybook
-   [x] I added a screenshot of it working
-   [x] UX has reviewed and approved this
-   [x] I assigned this PR to someone on the dev team to review
-   [x] I used a conventional commit message
-   [x] I assigned myself to this PR

Screenshot 2024-07-17 at 8 25 46 PM

[APPS-2799]: https://uclalibrary.atlassian.net/browse/APPS-2799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib-components/BlockScreeningDetail.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      />
+    </div>
+
+    <RichText
+      v-if="text"
+      class="text"
+      :rich-text-content="text"
+    />
+
+    <VideoEmbed
```
Comment (farosFreed) [inline src/lib-components/BlockScreeningDetail.vue:158]: The VideoEmbed component is not responsive on its own, it depends on responsiveVideo for the behavior. 
Please see the ReponsiveVideo.stories file for an example of how to slot VideoEmbed into RepsonsiveVideo for responsive behavior

**Review by farosFreed — CHANGES_REQUESTED** [review]
Mostly looks good, left a comment about how to get responsive videoembed

**Review by jendiamond — COMMENTED** [review]
🍰

**Review by farosFreed — APPROVED** [review]
Looks great!

**Pull request merged**